### PR TITLE
Add operator image workflow

### PR DESCRIPTION
**Pull request by Vincent056**
This PR switches the GitHub workflow to listen to the master branch so that we get a new operator image on any new commits on the master branch. We also added jobs to build bundle and openscap images to the same workflow. Also, we added a special bundle image dockerfile, so that the bundle image can point to the correct operator image and openscap image.

### CODE REVIEW

**Review by jhrozek — APPROVED** [review]
/lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Fix openscap image substitution in Makefile so that the correct image is used.
 
+- Fix github operator image workflow to listen to the master branch, added
+  bundle image, and openscap image build jobs to the workflow. Also added
+  a special Dockerfile for the bundle image so that the bundle image can
+  point to the correct images.
```
Comment (rhmdnd) [inline CHANGELOG.md:25]: This is all primarily to improve CI coverage, right?

Comment (Vincent056) [reply CHANGELOG.md:25]: This is for CI to use master image build, so we don't have to do an upstream release if something breaks in CI.

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Comment by Vincent056** [comment]
cluster issue, retest

**Comment by rhmdnd** [comment]
/retest-required

Timeout issue with AWS, which we're seeing across several patches.

**Pull request merged**